test/biased_coin.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats
+
+@test "${BATS_TEST_FILENAME##*/}: Integration test for learning biased coins" {
+    # the factor graph used for test is from biased coin, which contains 18
+    # variables,
+    # 1 weight, 18 factors, and 18 edges. Variables of id 0-8 are evidence: id 0-7
+    # positive and id 8 negative.
+    fg="$BATS_TEST_DIRNAME"/coin
+
+    # generate factor graph from tsv
+    ./text2bin variable "$fg"/variables.tsv "$fg"/graph.variables
+    ./text2bin weight   "$fg"/weights.tsv   "$fg"/graph.weights
+    ./text2bin factor   "$fg"/factors.tsv   "$fg"/graph.factors 4 1 0 1
+
+    # run sampler
+    ./dw gibbs \
+        -w "$fg"/graph.weights \
+        -v "$fg"/graph.variables \
+        -f "$fg"/graph.factors \
+        -m "$fg"/graph.meta \
+        -o "$fg" \
+        -l 2000 -i 2000 -s 1 --alpha 0.1 --diminish 0.995 --sample_evidence --reg_param 0
+
+    # check results
+    cd "$fg"
```

Comment:
These repeated lines of test code seem to be good candidate for a helper function?  Can we turn these into a function that takes as input a factorgraph directory with some parameters and populates the result all in standard relative locations?